ibis/tests/expr/test_value_exprs.py

```diff
@@ -118,6 +119,7 @@ def test_literal_with_implicit_type(value, expected_type):
         (tuple(multipoint), 'multipoint'),
         (list(multipolygon1), 'multipolygon'),
         (tuple(multipolygon1), 'multipolygon'),
+        (uuid.uuid4(), 'uuid'),
```

Comment:
Can you also add a `(str(uuid.uuid4()), 'uuid')` case?

Reply:
> > @gforsyth I think you need to put the issue in the PR description otherwise GH won't pick it up for autoclosing.
> 
> Yeah, I hadn't seen the issue when I opened it and I don't think github picks up the autoclose if it gets edited into the PR description after the fact. I'll add it in just for the sake of summarization

Oh good, I was wrong!